data/json/items/tool/cooking.json

```diff
@@ -685,6 +685,13 @@
         "max_contains_volume": "20 L",
         "max_contains_weight": "20 kg",
         "item_restriction": [ "medium_battery_cell", "medium_plus_battery_cell", "medium_atomic_battery_cell", "medium_disposable_cell" ]
+      },
+      {
+        "pocket_type": "CONTAINER",
+        "max_contains_volume": "2 L",
+        "max_contains_weight": "4 kg",
+        "max_item_length": "35 cm",
+        "moves": 0
```

Comment:
Why is this zero?

Reply:
it's a temporary thing, idealy in case i find a way to preserve the old behavior. meaning that this pocket is only loaded and unloaded via the activation actions of the multi cooker but i am not sure that this is possible.